fabfile/db/__init__.py

```diff
@@ -169,17 +229,30 @@ def restore_db_postgres(self, suffix, *args, **kwargs):
         Restore Postgres DB.
         """
         with lcd(self.base_path), settings(hide('warnings'), warn_only=True):
-            if not local('ls %s/backups/backup.postgres.%s' % (self.base_path, suffix)).succeeded:
+            if not local('ls %s/backup.postgres.%s' %
+                        (self.backup_path, suffix)).succeeded:
                 return self.list_backups('postgres')
-            local('dropdb %s --username=%s -w ' % (self.db_cfg['NAME'], self.db_cfg['USER']))
+            local('dropdb %s_temp --username=%s -w ' % (self.db_cfg['NAME'],
+                                                        self.db_cfg['USER']))
```

Comment:
And of course such pass should be placed in pgpass and user should be passed into this local func - local('dropdb %s_temp --username=%s -w ' % (self.db_cfg['NAME'], self.user))